### PR TITLE
Fix swagger css

### LINKED ISSUE #253: OpenAPI docs don't handle dark mode well
Noticed that text colors do not update when dark mode is enabled:

Screenshot from 2023-03-26 15-26-01

### PR DESCRIPTION
**Pull request by szczys**
Swagger CSS has been broken, especially in dark mode. This PR fixes the style and closes #253 

## Before:
Screenshot 2023-11-25 at 22-52-24 Open API Docs Golioth

## After:
Screenshot 2023-11-25 at 22-52-37 Open API Docs Golioth

### CODE REVIEW

**Comment by szczys** [comment]
I spent some time trying to use an existing swagger-ui dark mode library but the issue is that Docusaurus is handling the dark mode switch, not the browser. All of the swagger themes I found use `prefers-color-scheme`:

https://github.com/Amoenus/SwaggerDark/blob/2064ccd45b571865a64c731fa6bfddfbf2a01fe1/SwaggerDark.user.css#L13

Because of that, if your system/browser prefers dark, but the Docusaurus toggle is set to light, it breaks the light mode styling.

Fixing it the way I have done here is brittle--a game of whack-a-mole where anything not anticipated will be styled incorrectly. But it will be hard to solve this correctly [unless proper support becomes available](https://github.com/swagger-api/swagger-ui/issues/5327).

**Comment by szczys** [comment]
I wonder if we could use this plugin to replace our openAPI section: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs

File: src/css/custom-swagger-ui.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  background-color: var(--golioth-color-cod-gray);
+}
+
+html[data-theme="dark"] .swagger-ui .opblock-body pre.microlight {
+  background-color: var(--golioth-color-cod-gray) !important;
+}
+
+html[data-theme="dark"] .loading-container .loading {
+  filter: invert(100%);
+}
```
Comment (hasheddan) [inline src/css/custom-swagger-ui.css:101]: Noticed this one is not restricted to the `.swagger-ui` class -- is that intentional?

Comment (szczys) [reply src/css/custom-swagger-ui.css]: Good catch, I've added and tested that change as working.

**Review by hasheddan — APPROVED** [review]

**Pull request merged**